packages/perseus/src/widgets/radio/multiple-choice-widget.new.tsx

```diff
@@ -71,278 +71,258 @@ type Props = WidgetProps<
     PerseusRadioRubric
 >;
 
-type DefaultProps = Required<
-    Pick<
-        Props,
-        | "choices"
-        | "multipleSelect"
-        | "countChoices"
-        | "deselectEnabled"
-        | "linterContext"
-        | "showSolutions"
-    >
->;
-
 /**
  * MultipleChoiceWidget implements the Widget interface for multiple choice questions.
  *
- * It handles Perseus-specific logic and state management while delegating
+ * It handles Perseus-specific logic and user state management while delegating
  * UI rendering to the MultipleChoiceComponent.
  *
- * This class is exported as "Radio" for backwards compatibility with existing content,
+ * This component is exported as "Radio" for backwards compatibility with existing content,
  * though it supports both radio-button (single select) and checkbox (multiple select) modes.
  * Eventually, when Content Backfills are set up, we will officially rename the Radio Widget to MultipleChoiceWidget.
  *
  * Created as part of the Radio Revitalization Project (LEMS-2933).
  */
-class MultipleChoiceWidget extends React.Component<Props> implements Widget {
-    static defaultProps: DefaultProps = {
-        choices: [],
-        multipleSelect: false,
-        countChoices: false,
-        deselectEnabled: false,
-        linterContext: linterContextDefault,
```

Comment:
We were setting a default for linterContext...even though we're hardcoding the only place we're using it. (Line 162) Given that, I've removed the default. 

We also aren't using deselectEnabled anymore.